### DISCUSSION TITLE
Clearer Documentation on filtering and sorting

### DISCUSSION THREAD

**Discussion by t-lark** [CrowdStrike/falconpy#375] [Q&A] [answered]
**Describe the bug**
Not a bug, also thank you for building this. As someone who knows Python decently, this made the barrier to entry to the CS API pretty easy.  However, I was quite confused on how the syntax for filtering 

**To Reproduce**
none

**Expected behavior**
more examples in code of how to use filtering and sorting 

**Environment (please complete the following information):**
n\a

**Additional context**
It took me an hour or two of reading docs, github issues, and general google searching to figure out I had to use the following example 

```
PARAMS = {
    'offset': 0,
    'limit': 100,
    'sort': 'version.desc',
    'filter': 'os: "Amazon Linux" + os_version: "2"'
}
```

the `+` operator instead of the SQL like of AND or OR which is mentioned in the official CS docs

**Comment by jshcodes** [comment, marked as answer]
This documentation is in flight, you should start seeing updates show up in the [wiki](https://github.com/CrowdStrike/falconpy/wiki) this week. (This will take a week or two to complete.) 

Also starting with v0.7.0+ you will see some of this detail reflected within docstrings. (Making them accessible to you with help(className))
![Screen Shot 2021-10-12 at 10 09 35 PM](https://user-images.githubusercontent.com/74007258/137054812-dfdc1b0c-124a-43fe-a8b4-db4fcff1b705.png)

Example of an updated page reflecting the parameters you show above: [Hosts Service Collection Documentation](https://github.com/CrowdStrike/falconpy/wiki/Hosts)

FQL documentation can also be found here: [Falcon Query Language](https://github.com/CrowdStrike/falconpy/wiki/Falcon-Query-Language)

Thank you for the feedback, please let us know if you find any other issues! :smile:

**Comment by jshcodes** [comment]
Here's a better example of a docstring display showing filter / sort params.  
>Level of detail will vary by class, and will increase over time.

![Screen Shot 2021-10-12 at 10 30 48 PM](https://user-images.githubusercontent.com/74007258/137056912-c2802d30-de96-4f90-8aa9-edf963951d94.png)
